salty-trolls-react/src/components/authentication/Register.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import axios from "axios";
+
+class Register extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userLoginEmail: "",
+      userLoginPassword: "",
+      verifyPassword: ""
+    };
+  }
+
+  registerNewUser = () => {
+    if (this.state.userLoginPassword === this.state.verifyPassword) {
+      axios
+        .post("url", {
+          email: this.state.userLoginEmail,
+          password: this.state.userLoginPassword
+        })
+        .then(res => {
+          console.log(res);
+          localStorage.setItem("token", res.data.token);
+        })
+        .catch(err => {
+          console.log(err.message);
+        });
+    } else {
+      return "Passwords do not match.";
```

Comment:
Great work testing if the password and verification match before sending to server. This is exactly what I was asking for!